src/components/getOrganizationLinks.js

```diff
@@ -6,21 +6,22 @@ const getGithubInfo = (organization) => {
   if (links && links.length) {
     const githubLink = links.find((link) => link.link_type === 'GitHub');
     if (githubLink) {
-      const githubId = organization.github_id;
-
-      info.imageUrl = githubId
-        ? `https://avatars1.githubusercontent.com/u/${githubId}?s=100&v=4`
-        : organization.image_url;
       info.organizationUrl = githubLink.url;
     }
   }
   return info;
 };
 
 const getImageFromOrg = (organization) => {
-  if (organization.image_url && !organization.image_url.includes('scontent')) {
+  const githubId = organization.github_id;
+  if (githubId !== null) {
+    return `https://avatars1.githubusercontent.com/u/${githubId}?s=100&v=4`;
+  }
+  if (organization.image_url.includes('wp-content'))
+  {
     return organization.image_url;
-  } else {
+  }
+  else {
```

Comment:
since there are return statements above, do we need this else statement here?  It seems like we can just call `return '/images/default-github-repo-image.png';` at the bottom of all this logic

Reply:
There is a some issue for Code for ALL , The github id is null as well as the image url link is giving error. So If we remove that statement there will be no image displayed for Code for ALL

Reply:
@bhaggya  I only removed the else statement (not the return statement).  Could you pull down the latest orgsicon-on-local to your local branch and check if my changes are acceptable?

Reply:
Yes it is acceptable, it is working fine